packages/tasks/src/hardware.ts

```diff
@@ -164,7 +224,7 @@ export const SKUS = {
 				tflops: 21.75,
 				memory: [8],
 			},
-			"RTX 3070 Ti Laptop": {
```

Comment:
Btw this broke some users - resulting in 500s on HF due to discrepancy between DB data & code data (not a big deal)